nursery/empty-recycle-bin-quiet.yml

```diff
@@ -0,0 +1,16 @@
+rule:
+  meta:
+    name: empty recycle bin quiet
+    namespace: host-interaction/recycle-bin
+    author: matthew.williams@fireeye.com
+    scope: basic block
+    references:
+      - https://docs.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shemptyrecyclebina
+    examples:
+      - 276F691A3DF25481F59D79781799E35F:0x1400254E0
+  features:
+    - and:
+      - api: SHEmptyRecycleBin
+      - or:
+        - offset: 7 = SHERB_NOSOUND|SHERB_NOPROGRESSUI|SHERB_NOCONFIRMATION
```

Comment:
I like that you cover all bases here. Did you observe the usage of an offset for these constants in a sample?

Reply:
Yeah, the example I provided is 64-bit and uses `lea` to populate the "quiet" argument:
```asm
xor     edx, edx        ; pszRootPath
xor     ecx, ecx        ; hwnd
lea     r8d, [rdx+7]    ; dwFlags
call    cs:SHEmptyRecycleBinW
```

Reply:
Thank you, I've opened this up for further discussion in https://github.com/fireeye/capa/issues/320. I'll hold off merging for now.